jobs/build-triggers/tendrl-build-triggers-pr-tests.yml

```diff
@@ -0,0 +1,173 @@
+    builders:
+      - shell: |
+          #!/bin/bash -xe
+          echo
+          if [[ "${{ghprbPullId}}" ]]; then
+            echo "ghprbPullId: ${{ghprbPullId}}"
+            echo "ghprbCommentBody: ${{ghprbCommentBody}}"
+            echo "ghprbPullLongDescription: ${{ghprbPullLongDescription}}"
+            ADDITIONAL_REPOS="${{ADDITIONAL_REPOS}},http://artifacts.ci.centos.org/tendrl/pr/{package_prefix}{package_name}/pr${{ghprbPullId}}/{package_prefix}{package_name}-pr${{ghprbPullId}}.repo"
+            echo
+            echo "Parse ghprbCommentBody and ghprbPullLongDescription for dependent PRs."
+            # filter only lines containing "depends on:" string (and remove it)
+            # so it will iterate over list similar to this:
+            # tendrl-notifier/pr153 tendrl-commons/pr820 ...
+            for dependency in $(echo -e "${{ghprbCommentBody}}\n${{ghprbPullLongDescription}}" | \
+                grep -i "depends on:" | sed 's/depends on: //I'); do
+              package=${{dependency%/*}}
+              pr_id=${{dependency#*/}}
+              ADDITIONAL_REPOS="${{ADDITIONAL_REPOS}},http://artifacts.ci.centos.org/tendrl/pr/${{package}}/${{pr_id}}/${{package}}-${{pr_id}}.repo"
+            done
+            echo
+          fi
+
+          # Prepare env.sh file
+          echo > ${{WORKSPACE}}/env.sh
+          # use cop tendrl/tendrl (master) repo
+          echo "TENDRL_REPO_URL=http://copr-be.cloud.fedoraproject.org/results/tendrl/tendrl/epel-7-x86_64/" >> ${{WORKSPACE}}/env.sh
+          echo "TENDRL_REPO_GPGKEY_URL=https://copr-be.cloud.fedoraproject.org/results/tendrl/tendrl/pubkey.gpg" >> ${{WORKSPACE}}/env.sh
+          echo "TENDRL_DEPENDENCIES_REPO_URL=http://copr-be.cloud.fedoraproject.org/results/tendrl/dependencies/epel-7-x86_64/" >> ${{WORKSPACE}}/env.sh
+          echo "TENDRL_DEPENDENCIES_REPO_GPGKEY_URL=https://copr-be.cloud.fedoraproject.org/results/tendrl/dependencies/pubkey.gpg" >> ${{WORKSPACE}}/env.sh
+          # configure additional repos
+          echo "ADDITIONAL_REPOS=${{ADDITIONAL_REPOS}}" >> ${{WORKSPACE}}/env.sh
+          echo "ADDITIONAL_REPOS_USM_SERVER=${{ADDITIONAL_REPOS_USM_SERVER}}" >> ${{WORKSPACE}}/env.sh
+          echo "ADDITIONAL_REPOS_USM_NODES=${{ADDITIONAL_REPOS_USM_NODES}}" >> ${{WORKSPACE}}/env.sh
+          echo "ADDITIONAL_REPOS_USM_CLIENT=${{ADDITIONAL_REPOS_USM_CLIENT}}" >> ${{WORKSPACE}}/env.sh
+          set +x
+
+          echo
+          echo "##############################################################"
+          echo "Run whole test suite."
+          echo
+      - trigger-builds:
+          - project: "tendrl-2-cluster-gluster"
+            property-file: env.sh
+            block: true
+
+    publishers:
+      - email:
+          recipients: dahorak@redhat.com
+
+
+- project:
+    name: 'Test_suite'
+    triggers:
+      - github-pull-request:
+          admin-list:
+            - dahorak
+          white-list:
```

Comment:
Is there a way to input this rather than hardcoding

Reply:
It's the same as for the build jobs. Itshould be possible, to automatically fetch the list of people from the organization specified below in the org-list section, but there seems to be some problem with this and I'm not sure why it doesn't work and where might be the problem with that.
I'll try to check that once more and find some more information about that - that will be definitely the best solution.